src/features/users/userApi.ts

```diff
@@ -6,27 +6,31 @@ import { UserImage, UserInfo, UserTheme } from 'types/user';
 export const userApi = createApi({
   reducerPath: 'userApi',
   baseQuery: fetchBaseQueryWithToken,
+  tagTypes: ['UserInfo', 'UserImage', 'UserTheme', 'UserNickname'],
   endpoints: (builder) => ({
     // builder.query<T, U>() --> T는 쿼리의 반환값 타입, U는 쿼리 파라미터의 타입.
     getUserInfo: builder.query<UserInfo, void>({
       query: () => ({
         url: '/user/info',
         method: 'GET',
       }),
+      providesTags: ['UserInfo', 'UserImage', 'UserNickname', 'UserTheme'],
```

Comment:
```suggestion
      providesTags: ['UserInfo'],
```

이렇게 되어도 될듯?!

태그가 4개인 부분이 이해가 안돼서 직접 여쭤봄 ㅎㅎ